holiday-calendar-base/src/main/java/com/github/davejoyce/calendar/SpecialAnniversary.java

```diff
@@ -0,0 +1,118 @@
+/*******************************************************************************
+ * Holiday Calendar - A library for definition and calculation of holiday calendars
+ * Copyright (C) 2022 David Joyce
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option) any
+ * later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along
+ * with this library; if not, write to the Free Software Foundation, Inc., 59
+ * Temple Place, Suite 330, Boston, MA 02111-1307 USA
+ ******************************************************************************/
+
+package com.github.davejoyce.calendar;
+
+import java.time.LocalDate;
+import java.time.Month;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.StringJoiner;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * A holiday which commemorates a particular anniversary of an event. Example
+ * of a special anniversary is a Jubilee for a reigning monarch.
+ *
+ * @author <a href="mailto:dave@osframework.org">Dave Joyce</a>
+ */
+public class SpecialAnniversary extends Holiday {
+
+    private final LocalDate anniversaryDate;
+
+    public SpecialAnniversary(String name,
+                              String description,
+                              LocalDate anniversaryDate,
+                              boolean rollable) {
+        super(name, description, rollable);
+        this.anniversaryDate = requireNonNull(anniversaryDate, "Argument 'anniversaryDate' cannot be null");
+    }
+
+    public SpecialAnniversary(String name,
+                              String description,
+                              String anniversaryDate,
+                              boolean rollable) {
+        this(name, description, LocalDate.parse(anniversaryDate), rollable);
+    }
+
+    public LocalDate getAnniversaryDate() {
+        return anniversaryDate;
+    }
```

Comment:
`@Getter`?

Reply:
@battleblow will be part of the refactor in new issue.